purchase_requisition_incoterm/models/purchase_order.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    incoterm_id = fields.Many2one(
```

Comment:
Remove this field definition. Is already defined in `purchase_order_partner_incoterm`.